app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ReviewsController < ApplicationController
+  def index
+    @reviews = Reviews.find(params[:id])
+  end
+
+  def new
+    @review = Review.new
+  end
+
+  def create
+    @api = Api.find(params[:api_id])
```

Comment:
instance variables (`@`) are only required when the values are to be consumed in a view. As `create` does not go to a view, consider making these traditional, non-instance variables.